apps/andi/config/dev.exs

```diff
@@ -19,6 +19,9 @@ config :andi, AndiWeb.Endpoint,
       "--watch-stdin",
       cd: Path.expand("../assets", __DIR__)
     ]
+  ],
+  live_view: [
+    signing_salt: "TOP SECRET!!!"
```

Comment:
Should we just put this value in `config.exs` and remove it from these environment-specific files? It looks like the actual value isn't really important to us yet.

Reply:
I don't think so because it needs to be set in each environment, but only in Prod does it need to be a secret